protocol/contracts/reserve/src/api.rs

```diff
@@ -26,9 +26,14 @@ pub struct MigrateMsg {}
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
 #[serde(deny_unknown_fields, rename_all = "snake_case")]
 pub enum ExecuteMsg {
-    NewLeaseCode { code_id: Uint64 },
+    /// This is an internal system API and we use [Code]
+    NewLeaseCode {
+        code: Code,
+    },
 
-    CoverLiquidationLosses { amount: LpnCoin },
+    CoverLiquidationLosses {
+        amount: LpnCoin,
+    },
```

Comment:
[Suggestion] We could simplify the serialized version if we use a a tuple variant.
```suggestion
    CoverLiquidationLosses(LpnCoin),
```
This would produce `{"cover_liquidation_losses": "1000unls"}` in JSON instead of nested objects.